### ISSUE TITLE
table captions should be first descendant of table tag

### ISSUE THREAD

**Issue by simonista** [tinymce/tinymce#3034]
Hey All,  according to the HTML spec, the `caption` tag should be the first descendant of the `table` tag. (see https://www.w3.org/TR/html5/tabular-data.html#the-caption-element).  Webaim mentions this as being important for screenreaders to be able to properly understand tables: http://webaim.org/techniques/tables/data#caption

Steps to reproduce:
- create a table
- go to table properties and check the "caption" button
- enter a caption
- make the first row a header row (table -> row -> row properties -> row type)
- inspect the html and notice that the `thead` comes before the `caption`

**Comment by WebVeteran** [comment]
I have a fix for this in #3319

**Comment by simonista** [comment]
Hey @spocke, sorry to ping again on this, is there anything I can do to help get the associated Pull Request merged?

**Comment by jayarjo** [comment]
I will look into this this week.

**Comment by bkirkby** [comment]
@jayarjo any word on this? i'd really like to get this piece compliant with WebAIM accessibility standards so i can use it. thanks in advance.

**Comment by fyrkant** [comment]
This bug/issue has been fixed in the latest version. If you can still reproduce this on the latest version please let us know and we re-open this ticket. Thanks.
